src/Pagination.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
+import { connect } from 'react-redux'
+import { changePage } from './actions/paging'
 
-export default class Pagination extends React.Component {
+export class Pagination extends React.Component {
   constructor(props) {
     super(props);
     this.state = this._calculatePages(props);
```

Comment:
Moving forward with redux might we want to store page numbers, etc as part of the state tree?